lib/ash.ex

```diff
@@ -3814,25 +3814,130 @@ defmodule Ash do
     use Spark.Options.Validator, schema: transaction_opts
   end
 
+  @doc deprecated: """
```

Comment:
These docs are strangely combined. The `@doc deprecated: ...` should be separate from the function docs.

Reply:
Ah, I wasn't sure about the syntax and the docs don't show an example with both deprecated and regular docs. Do I just add another @doc before the function or how does that work?

Reply:
Ohh I see what you were going for. Yeah its `@doc deprecated: "string"` and `@doc "the docs"` 👍